lib/measured/unit_system_builder.rb

```diff
@@ -8,12 +8,36 @@ def unit(unit_name, aliases: [], value: nil)
     nil
   end
 
+  def si_unit(unit_name, aliases: [], value: nil, range: range = 3)
+    @units.concat build_si_units(unit_name, aliases: aliases, value: value, range: range)
```

Comment:
We prefer using parentheses for all function calls except for DSL-like things. Another option is `@units += ...`